src/vstart.sh

```diff
@@ -935,11 +935,13 @@ EOF
     fi
 
     if [ "$objectstore" == "seastore" ]; then
-      if [[ ${seastore_size+x} ]]; then
-        seastore_size=$(echo "$seastore_size*1024^3/1" | bc)
-        SEASTORE_OPTS="
-        seastore device size = $seastore_size"
+      if [ -z ${seastore_size} ]; then
+        # fallback to 10GB
+        seastore_size=10
```

Comment:
Was the default size 50G too large ?

Reply:
Starting a local vstart cluster with 4-5 OSDs will require too much space for a dev machine. In any way, I'll close this PR since the vstart `--seastore-device-size` can be used instead.